grammar.js

```diff
@@ -1327,14 +1353,16 @@ module.exports = grammar({
           $.keyword_overwrite, // Spark SQL
         ),
       ),
-      $.table_reference,
+      $.object_reference,
       optional($.table_partition), // Spark SQL
       optional(
         seq(
           $.keyword_as,
-          field('table_alias', $._alias_identifier)
+          field('alias', $.identifier)
         ),
       ),
+      // TODO i don't think we have a test for `insert into tbl set ....` and
+      // it's no dialect i can think of
```

Comment:
If I read the docs correclty, then MariaDB allows for this:
https://mariadb.com/kb/en/insert/